beard_server/__init__.py

```diff
@@ -1,32 +1,36 @@
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio.
-# Copyright (C) 2015 CERN.
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
 #
-# Invenio is free software; you can redistribute it
+# Inspire is free software; you can redistribute it
 # and/or modify it under the terms of the GNU General Public License as
 # published by the Free Software Foundation; either version 2 of the
 # License, or (at your option) any later version.
 #
-# Invenio is distributed in the hope that it will be
+# Inspire is distributed in the hope that it will be
 # useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 # General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with Invenio; if not, write to the
+# along with Inspire; if not, write to the
 # Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
 # MA 02111-1307, USA.
 #
 # In applying this license, CERN does not
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Application providing Beard as a Celery service."""
+"""Beard as RESTful API and Celery service."""
 
-from __future__ import absolute_import, print_function
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals)
```

Comment:
You're going to hate me for this, but @jirikuncar suggested that we should drop `unicode_literals` from the list of imports, since it can cause all sorts of hard-to-debug issues.